### ISSUE TITLE
Unrecognised model in goland test case

### ISSUE THREAD

**Issue by AleSgueglia** [ImperialCollegeLondon/sharpy#143]
Hello,
I just started playing with SHARPy, and to start I downloaded the Goland wing test case and run it on my machine (using the Docker application). 
During the simulation, some warning occured of unrecognised setting: 
 ```
Unrecognised setting: rigid_body_motion. Please check input file and/or documentation
Unrecognised setting: use_euler. Please check input file and/or documentation
Unrecognised setting: compute_fom. Please check input file and/or documentation.
```
I checked on the webpage and those messages don't appear in the simulation results mentioned on the doc, so I would like to ask if there is something I am missing. 
For completeness I attach the whole messages out of the simulation. 

Best regards,
Alessandro Sgueglia
[goland_log.log](https://github.com/ImperialCollegeLondon/sharpy/files/6350777/goland_log.log)

**Comment by ngoiz** [comment]
Hi Alessandro,

Thanks for raising this. 

This is a result of us changing the input settings and forgetting to update the input files accordingly...the simulation will run fine, since it is just ignoring legacy settings that were previously used and that no longer affect the solution process.

I will update it but in the mean time you can keep running it as is. If you encounter more of these warnings (you very well may) just ignore them. We are going to turn them into errors so hopefully we'll get rid of all those for the next stable release.

Thanks and best regards,
Norberto

**Comment by AleSgueglia** [comment]
Hello Norberto,
thanks for your quick reply. Indeed I imagined that you were implementing new features and the doc is still based on older version, and I'll wait for the updated version.
In the meantime, is it possible to know at which those inputs now correspond? Because I would like to start to run my own flutter analyses and I think could be useful to have a full comprehension of all the setup. 

Thanks again and best regards, 
Alessandro

**Comment by ngoiz** [comment]
Hi Alessandro,

These are the updated Goland test cases (the actual test and the Jupyter notebook): 
https://github.com/ImperialCollegeLondon/sharpy/blob/dev_setting_error/docs/source/content/example_notebooks/linear_goland_flutter.ipynb

https://github.com/ImperialCollegeLondon/sharpy/blob/dev_setting_error/tests/linear/goland_wing/test_goland_flutter.py

I just tested that these will both work on the released version of SHARPy and should not contain any deprecated settings.

You have a full description of what the settings are in the docs in the [solvers](https://ic-sharpy.readthedocs.io/en/master/content/solvers.html) page and for the [linear modules](https://ic-sharpy.readthedocs.io/en/master/includes/linear/assembler/index.html) used in the `LinearAssembler` solver.

Hope this helps,
Norberto

**Comment by ngoiz** [comment]
This has been fixed in #148. Deprecated/unrecognised settings raise an error and all examples and templates have been updated to reflect the latest settings. It will be available at the next SHARPy release, for the moment, available in the `develop` branch.
